pyproject.toml

```diff
@@ -0,0 +1,37 @@
+[build-system]
+requires = ["hatchling", "hatch-vcs"]
+build-backend = "hatchling.build"
```

Comment:
OK we'll give hatchling a try

Some justification they provide for how it might be better; though I know some people like source builds to ship with testing suites - https://hatch.pypa.io/latest/plugins/builder/wheel/#default-file-selection

Reply:
Yeah whatever is fine. Just saw that used in the Zarr project